Address asciidoctor-dita-vale issues in Administering Satellite (part 2) (#4527)
theforeman/foreman-documentation: guides/common/modules/proc_monitoring-puppet.adoc

[id="Monitoring_Puppet_{context}"]
= Monitoring Puppet

[role="_abstract"]

> You can monitor Puppet activity and certificate management through the Smart Proxy interface.

Reviewer: ```suggestion
You can monitor Puppet activity and certificate management through the {SmartProxy} interface.
```

I am a bit unsure. What do you think about "... management for each {SmartProxy}."? Or "... management on a per-{SmartProxy} basis."?

> This helps you verify that Puppet services run correctly and troubleshoot any issues.

Reviewer: IMO it does not. It only shows the current state of Puppet, aka. Puppet envs and class per env.

If you turn off `puppet` and `puppetserver` services on Smart Proxies, the WebUI will still show them but also show an error message:

````
Failure: ERF50-5345 [Foreman::WrappedException]: Unable to connect ([ProxyAPI::ProxyException]: ERF12-5356 [ProxyAPI::ProxyException]: Unable to get PuppetCA certificates ([RestClient::NotAcceptable]: 406 Not Acceptable) for proxy https://foreman.example.com:9090/puppet/ca)
````

It will not show any information about running services. I suggest to drop line 8:

```suggestion
```

[NOTE]
====
The *Puppet* and *Puppet CA* tabs are available only if you have Puppet enabled in your {Project}.
====

.Procedure
. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and click the name of the selected {SmartProxy}.
* At the *Puppet* tab you can find the following:
** A summary of Puppet events, an overview of latest Puppet runs, and the synchronization status of associated hosts at the *General* sub-tab.
** A list of Puppet environments at the *Environments* sub-tab.
* At the *Puppet CA* tab you can find the following:
** A certificate status overview and the number of autosign entries at the *General* sub-tab.
** A table of CA certificates associated with the {SmartProxy} at the *Certificates* sub-tab.
Here you can inspect the certificate expiry data, or cancel the certificate by clicking *Revoke*.
** A list of autosign entries at the *Autosign entries* sub-tab.
Here you can create an entry by clicking *New* or delete one by clicking *Delete*.

.Additional resources

> * {ManagingConfigurationsPuppetDocURL}Enabling_Puppet_Integration_managing-configurations-puppet[Enabling Puppet Integration with {Project} in _{ManagingConfigurationsPuppetDocTitle}_]

Reviewer: ```suggestion
* {ManagingConfigurationsPuppetDocURL}Enabling_Puppet_Integration_managing-configurations-puppet[Enabling Puppet integration with {Project} in _{ManagingConfigurationsPuppetDocTitle}_]
```
